bin/promotion.ml

```diff
@@ -0,0 +1,5 @@
+open Import
+
+let info = Cmd.info "promotion"
+
+let group = Cmd.group info [ Promotion_run.command; Promotion_diff.command ]
```

Comment:
There's a lot of `Promot*` modules in bin/ that don't seem particularly useful individually. How about a single `Promotion` module that exports all the relevant commands (for both `dune promote` and `dune promotion`). That should give a smaller API and be easier to understand.

Reply:
good idea, done